patch-package
--- c/node_modules/ethereum-waffle/dist/utils.js
+++ i/node_modules/ethereum-waffle/dist/utils.js
@@ -35,8 +35,7 @@ var eventParseResultToArray = exports.eventParseResultToArray = function eventPa
 };
 
 var isWarningMessage = exports.isWarningMessage = function isWarningMessage(error) {
-  return (/: Warning: /.test(error)
-  );
+  return error.severity === 'warning';
 };
 
 var getBalanceChange = exports.getBalanceChange = function () {
